components/navbar/SideNav.js

```diff
@@ -64,7 +67,7 @@ export default function SideNav({ navigation }) {
                               href={subItem.href}
                               className={classNames(
                                 subItem.href.toLowerCase() == pathname &&
-                                  "text-secondary-medium dark:!text-blue-500 font-semibold",
+                                "text-secondary-medium dark:!text-blue-500 font-semibold",
```

Comment:
this indentation is required by Prettier, please add the VScode Prettier plugin to automatically fix these

```suggestion
                                  "text-secondary-medium dark:!text-blue-500 font-semibold",
```